### ISSUE TITLE
Vector graphics output 

### ISSUE THREAD

**Issue by MKLau** [CamFlow/CamFlowR#17]
See https://github.com/End-to-end-provenance/DDG-Explorer/issues/81

**Comment by tfjmp** [comment]
Cytoscape people have/had plan for SVG support, but not sure if it is actively being worked on.

See: https://github.com/cytoscape/cytoscape.js/issues/639

If it ever come to fruition, that should prove no issue to provide a scalable vector export. At the moment we are limited to jpeg and png.

**Comment by MKLau** [comment]
Yeah, looks like they might have already done it: https://github.com/cytoscape/cytoscape.js/issues/639#issuecomment-275120784
